src/client/statics/ExitPage.js

```diff
@@ -13,6 +14,14 @@ export default class ExitPage extends React.Component {
     location: PropTypes.shape().isRequired,
   };
 
+  componentWillMount() {
+    const url = decodeURIComponent(new URLSearchParams(location.search).get('url'));
+    const hostname = new URL(url).hostname;
+    if (whitelistDomains.includes(hostname.replace(/^[^.]+\./g, ''))) {
```

Comment:
`includes` doesn't work in IE11 so it will crash Busy.

Reply:
no polyfills are used ?

Reply:
instead of modifying the exitPage may be I should test the whitelist before and use the `secureLinks` options ?
Sorry I'm still discovering the app tbh :)

Reply:
Yes it would be better to see the link from the post directly and make direct link to it. Perhpas we can have a component "ExternalLink" where we put the logic.

Reply:
We should have polyfills (via `babel-polyfill`), but for some reason exit page crashes on IE11 on Windows 10 (works on staging). I'm not sure what is the issue as BrowserStack doesn't open dev tools for me, and I don't have any windows machine right now.